conda_lock/conda_lock.py

```diff
@@ -372,6 +375,23 @@ def parser():
         action="append",
         help="generate lock files for the following platforms",
     )
+    parser.add_argument(
+        "--override-channels",
+        action="store_true",
+        help="""
+            Do not use channel information from source files.  This should be used in
+            conjunction with the -c/--channel flag to specify the channels desired
+        """,
+        default=False,
+    )
+    parser.add_argument(
+        "-c",
+        "--channel",
+        nargs="?",
+        action="append",
+        help="Additional channels to include in the solve.  Requires the --override-channels flag",
+    )
```

Comment:
I think you need a default value here too or args.channels is gonna raise down on line 469

Reply:
changed the channels to allow None